labs/lab-08/guides/registers-terminator/support/registers_terminator.asm

```diff
@@ -0,0 +1,53 @@
+%include "../utils/printf64.asm"
+
+section .data
+    msg db 'Hello, world!', 10, 0
```

Comment:
```suggestion
    msg db 10, 'printf called now', 10, 10, 0
```
This serves to better highlight the cut-off point before and after the registers should be changed.